src/live.rs

```diff
@@ -36,19 +36,29 @@ const COREOS_KARG_EMBED_AREA_HEADER_SIZE: u64 = 72;
 const COREOS_KARG_EMBED_AREA_HEADER_MAX_OFFSETS: usize = 6;
 const COREOS_KARG_EMBED_AREA_MAX_SIZE: usize = 2048;
 
-pub fn iso_embed(config: &IsoIgnitionEmbedConfig) -> Result<()> {
+pub fn iso_embed(config: &IsoEmbedConfig) -> Result<()> {
     eprintln!("`iso embed` is deprecated; use `iso ignition embed`.  Continuing.");
-    iso_ignition_embed(config)
+    iso_ignition_embed(&IsoIgnitionEmbedConfig {
+        force: config.force,
+        ignition_file: config.config.clone(),
+        output: config.output.clone(),
+        input: config.input.clone(),
+    })
```

Comment:
Can't the `IsoCmd` enum just directly use the same `IsoIgnition*Config` structs to avoid this?

Reply:
That was my plan initially, but `iso ignition embed` renamed `-c`/`--config` to `-i`/`--ignition-file`.  We could use `alias`, but that would allow both subcommands to accept both arguments and would mess up the help text for one of them.

After some thought, I think the approach here is actually better, because it clearly separates the semantics of the old and new subcommands.  This way any options added to the new subcommands will not be reflected in the old ones.